### PR TITLE
MR-4: session

### PR DESCRIPTION
**Pull request by KirillSmirnow**

### CODE REVIEW

File: server/src/main/kotlin/org/itmodreamteam/myrest/server/service/user/UserServiceImpl.kt
Comment (KirillSmirnow) [reply server/src/main/kotlin/org/itmodreamteam/myrest/server/service/user/UserServiceImpl.kt:6]: I don't care about imports in Java/Kotlin :rofl: . Imports are optimized by IDEA (Reformat Code).

File: server/src/main/kotlin/org/itmodreamteam/myrest/server/model/user/Session.kt
Comment (Rznov) [inline server/src/main/kotlin/org/itmodreamteam/myrest/server/model/user/Session.kt:42]: Just import randomUUID method:)

File: server/src/main/kotlin/org/itmodreamteam/myrest/server/model/user/Session.kt
Comment (Rznov) [inline server/src/main/kotlin/org/itmodreamteam/myrest/server/model/user/Session.kt:44]: Why such a time span?

File: shared/src/commonMain/kotlin/org/itmodreamteam/myrest/shared/user/Profile.kt
Comment (Rznov) [inline shared/src/commonMain/kotlin/org/itmodreamteam/myrest/shared/user/Profile.kt:9]: How are we gonna set the role?

**Review by Rznov — CHANGES_REQUESTED** [review]

File: server/src/main/kotlin/org/itmodreamteam/myrest/server/model/user/Session.kt
Comment (KirillSmirnow) [reply server/src/main/kotlin/org/itmodreamteam/myrest/server/model/user/Session.kt:44]: Long enough so that user doesn't have to sign in each time)) What do you think? Long? Short?

File: shared/src/commonMain/kotlin/org/itmodreamteam/myrest/shared/user/Profile.kt
Comment (KirillSmirnow) [reply shared/src/commonMain/kotlin/org/itmodreamteam/myrest/shared/user/Profile.kt:9]: Setting the role of system administrator can be done manually via database :)
OR we can automatically create this user at start up if it doesn't exist.

**Comment by KirillSmirnow** [comment]
Cannot merge without explicit approval

**Pull request closed without merging**
